Packs/DeveloperTools/Integrations/CustomIndicatorDemo/Integrations/CustomIndicatorDemo/CustomIndicatorDemo.py

```diff
@@ -0,0 +1,136 @@
+import demistomock as demisto
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
+
+import requests
+import traceback
+from typing import Dict, Any
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()  # pylint: disable=no-member
+
+
+''' CLIENT CLASS '''
+
+
+class Client(BaseClient):
+    """Client class to interact with the service API
+
+    This Client implements API calls, and does not contain any XSOAR logic.
+    Should only do requests and return data.
+    It inherits from BaseClient defined in CommonServer Python.
+    Most calls use _http_request() that handles proxy, SSL verification, etc.
+    For this  implementation, no special attributes defined
+    """
+
+    def baseintegration_dummy(self, dummy: str) -> Dict[str, str]:
+        """Returns a simple python dict with the information provided
+        in the input (dummy).
+
+        :type dummy: ``str``
+        :param dummy: string to add in the dummy dict that is returned
+
+        :return: dict as {"dummy": dummy}
+        :rtype: ``str``
+        """
+
+        return {"dummy": dummy}
+
+
+def test_module(client: Client) -> str:
+    """Tests API connectivity and authentication'
+
+    Returning 'ok' indicates that the integration works like it is supposed to.
+    Connection to the service is successful.
+    Raises exceptions if something goes wrong.
+
+    :type client: ``Client``
+    :param Client: client to use
+
+    :return: 'ok' if test passed, anything else will fail the test.
+    :rtype: ``str``
+    """
+    try:
+        message = 'ok'
+    except DemistoException as e:
+        if 'Forbidden' in str(e) or 'Authorization' in str(e):
+            message = 'Authorization Error: make sure API Key is correctly set'
+        else:
+            raise e
+    return message
+
+
+def test_custom_indicator(client: Client) -> CommandResults:
+    # Command using a custom indicator example
+
+    result = client.baseintegration_dummy("test")
+    score = Common.DBotScore.GOOD
+    indicator_value = 'custom_value'
+
+    # Create a DBotScore object
+    # Give it an indicator_type of DBotScoreType.CUSTOM
+    dbot_score = Common.DBotScore(
+        indicator=indicator_value,
+        indicator_type=DBotScoreType.CUSTOM,
+        integration_name='DummyIntegration',
+        score=score
```

Comment:
[PEP8](https://www.python.org/dev/peps/pep-0008/#when-to-use-trailing-commas)
```suggestion
        score=score,
```

Reply:
fixed